src/main/java/sx/blah/discord/handle/impl/obj/Channel.java

```diff
@@ -579,7 +584,7 @@ public boolean equals(Object other) {
 	 *
 	 * @param typingStatus The new typing status.
 	 */
-	public void setTypingStatus(boolean typingStatus) {
+	public void setCachedTypingStatus(boolean typingStatus) {
```

Comment:
You could probably get away with removing this method altogether as its mirrored in line 278